samples/bluetooth/mesh/sensor_server/sample.yaml

```diff
@@ -4,9 +4,12 @@ sample:
 tests:
   samples.bluetooth.mesh.sensor_server:
     build_only: true
-    platform_allow: nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf21540dk_nrf52840
+    platform_allow: nrf52dk_nrf52832 nrf52840dk_nrf52840 thingy53_nrf5340_cpuapp thingy53_nrf5340_cpuapp_ns
```

Comment:
I was going to, but the on-chip temerature sensor of the 53DK is part of the network core. I was not able to find a way to extract the sensor data from this chip, and it seems like there a no example that shows how this can be achived. To be able to add 53DK to the list of targets I would need assistance from somebody that knows how to acheive this, which I for now has not been able to find.